linkerd/app/outbound/src/http/concrete.rs

```diff
@@ -119,7 +123,20 @@ impl<N> Outbound<N> {
                 )
                 .instrument(|e: &Endpoint<T>| info_span!("endpoint", addr = %e.addr));
 
+            let mk_breaker = |target: &Balance<T>| {
+                match target.parent.param() {
+                    FailureAccrual::None => |_: &(SocketAddr, Metadata)| {
+                        // No failure accrual for this target; construct a gate
+                        // that will never close.
+                        tracing::trace!("No failure accrual policy enabled.");
```

Comment:
Since this made me double check the gate's behavior, it may be worth

```suggestion
                    FailureAccrual::None => |_: &(SocketAddr, Metadata)| {
                        // Construct a gate channel, dropping the controller
                        // side of the channel such that response summaries
                        // are never read. The failure accrual gate never
                        // closes in this configuration.
                        tracing::trace!("No failure accrual policy enabled");
```

(Also note that we virtually never use punctuation in log messages)